build-system/tasks/css/jsify-css.js

```diff
@@ -137,7 +137,7 @@ async function transformCss(contents, hash, opt_filename) {
   }
 
   const transformed = transform(contents, opt_filename);
-  transformCache.set(
+  await transformCache.set(
```

Comment:
similar nit to my first comment. I don't think this `await` is necessary

Reply:
This has now been reverted.